src/models/mSeriesChallenge.ts

```diff
@@ -27,8 +27,8 @@ export const enum MSeriesChallengeSorting {
 }
 
 export interface MSeriesChallengeData {
-  // joinCode: string | undefined
   id: number
+  joinCode: string
```

Comment:
IIRC, which I'm pretty sure I do because I checked with @ish2028 and he confirmed it; We said that it doesn't make sense to query/show a `MSeriesChallenge` by id unless you have a relationship to it(owned/joined). If a user tries to query/show a `MSeriesChallenge` by id without having a relationship to it, the API will reject it. Therefore, the only way to query/show a `MSeriesChallenge` that you have no relationship with, is to query it using the `joinCode`, and in that case the `joinCode` will be returned. So to answer your question, no, because the joinCode will always be returned.

Reply:
I don't agree, but I'll concede for the sake of getting this merged. This will likely cause issues once we implement the admin side of this so we'll have to address it then.